src/styles/components/fields/_field.scss

```diff
@@ -40,6 +40,7 @@
   }
   &__title {
     width: 30%;
+    //flex-shrink:0;
```

Comment:
This should be enabled. The titles should all be the same width, but if I keep this on, certain tabs will escape the boundaries of the content area of the field. @nicolaskruchten @VeraZab what do you two think of changing these fields that have the radio buttons (3+) into a label on top, and then a full width field widget?

flex-shrink: 1
<img width="315" alt="screen shot 2018-02-27 at 1 23 31 pm" src="https://user-images.githubusercontent.com/11803153/36749808-7a808cc0-1bc1-11e8-8040-17949c642a15.png">

flex-shrink: 0
<img width="357" alt="screen shot 2018-02-27 at 1 23 41 pm" src="https://user-images.githubusercontent.com/11803153/36749809-7a950ad8-1bc1-11e8-9f0b-3e35ebd92ac5.png">

Reply:
I'm OK with your proposal.